modules/@ergo/activity-logs/config/routes.js

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
+ * See LICENSE for license details.
+ */
+import { Pages, Tabs, Icons } from './imports';
+import Privileges from './privileges';
+
+export default [
+    {
```

Comment:
What about wrapping this objects into typed? Like... class Route which has name, path, component, class Meta etc.. ? I would be ez to manage and describe

Reply:
Each module should have an `routes.js` file with array export. I collect all routing files and combine them. Vue Router instance should have array as input. 
A simple format is enough, no class needed.
Doc: https://router.vuejs.org/api/#routes